docs/summary.rst

```diff
@@ -106,13 +106,23 @@ To install Horovod:
 
 3. Install the ``horovod`` pip package.
 
+   To run on CPUs:
+
 .. code-block:: bash
```

Comment:
You might want to indent lines 111 through 119 so they are formatted as a subsection of step 3